.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            4
+        ],
+        "quotes": [
+            2,
+            "single"
+        ],
+        "linebreak-style": [
```

Comment:
Let's remove linebreak styles on this one, too.

https://github.com/TheFourFifths/consus-flux/pull/9/commits/06e9f2992ede08b94495201fd3b35f72256d7e7d